print/test/package-lock.spec.js

```diff
@@ -0,0 +1,7 @@
+import { lockfileVersion } from '../package-lock.json'
+
+describe('The package-lock.json', () => {
+  test('uses lockFileVersion 1', () => {
+    expect(lockfileVersion).toBe(1)
```

Comment:
I assume running this one as well depends on #718?

Reply:
yes, it is not yet automatically run.